libres/tests/analysis/std_enkf.cpp

```diff
@@ -0,0 +1,39 @@
+#include <catch2/catch.hpp>
+
```

Comment:
Consider changing the name of the test-file to `test_std_enkf.cpp`.
We are basically following similar naming conventions as used in `pytest` (not always perfectly though).
Whether or not this is a good idea can be discussed, but it's nice if we can be somewhat consistent I think.

Reply:
Will do